### ISSUE TITLE
API/register broken

### ISSUE THREAD

**Issue by Cyberzoid1** [uidaho/smartsettia#29]
It looks like after the #27 was pushed the api no longer works.
Tried smartsettia.com/api/register as well your c9 workspace

```bash
curl -X POST https://smartsettia.com/api/register  -H "Accept: application/json"  -H "Content-Type: application/json"  -d '{"uuid": "4dbc0776-9b44-11e7-abc4-cec278b6b50a", "challenge": "temppass"}'^C
```

**Comment by Cyberzoid1** [comment]
It is returning the api html page

Also at some point I was receiving invalid json data with []
`[{"name":"bob","token":"123"}]`
instead of
`{"name":"bob","token":"123"}`

**Comment by brandonjank** [comment]
Fixed with commit 06cae1031c93938238976d7a7fb71f0ce7a1e00e
